v3/src/components/slider/slider-model.ts

```diff
@@ -53,11 +53,17 @@ export const SliderModel = TileContentModel
       }
       self.globalValue.setValue(n)
     },
-    setMultipleOf(n: number) {
-      self.multipleOf = Math.abs(n)
+    setMultipleOf(n: number | string) {
+      const multipleOf = typeof n !== "number" ? parseFloat(n) : n
+      if (!isNaN(multipleOf) && isFinite(multipleOf)) {
+        self.multipleOf = Math.abs(multipleOf)
+      }
     },
-    setAnimationRate(n: number) {
-      self.animationRate = Math.abs(n)
+    setAnimationRate(n: number | string) {
+      const rate = typeof n !== "number" ? parseFloat(n) : n
```

Comment:
This sort of input parsing should be the client's responsibility, not the model's.

Reply:
Turns out I didn't actually need them.